TbsCore/Tasks/SecondLevel/SendDeff.cs

```diff
@@ -72,9 +73,16 @@ public bool TroopsCountRecieved(Account acc, int[] troopsAtHome)
             }
 
             this.DeffAmount.Amount -= upkeepSent;
-            acc.Wb.Log($"Bot will send {upkeepSent} deff (in upkeep) from {this.Vill.Name} to {this.TargetVillage}. Still needed {this.DeffAmount.Amount} deff");
+            if (this.DeffAmount.Amount > 0)

```

Comment:
why is one warning other information but both send the same msg?

Reply:
hm one is enough, other is need more =))